it/itutil/WireMockHelper.scala

```diff
@@ -82,20 +82,36 @@ trait WireMockHelper {
     )
   }
 
-  def stubALBEGET(
-    postcode: String = IntegrationTestConstants.testPostCode.split(" ").mkString("+"),
-    filter: Option[String] = None,
-    expectedStatus: Int = 200,
-    addressJson: JsValue = AddressRecordConstants.addressRecordSeqJson): StubMapping = {
+  def stubGetAddressFromBEWithFilter(postcode: String = IntegrationTestConstants.testPostCode.split(" ").mkString("+"),
```

Comment:
Changing the name of this will break the tests in a few of the other files. Either need to revert or rattle the change through the rest of the tests.